components/campaign-hero/Component.jsx

```diff
@@ -0,0 +1,243 @@
+import React, { useState, useEffect, useRef } from "react";
+import { XssSafeContent } from "@squiz/xaccel-xss-safe-content";
+import Modal from "../../packages/modal/ModalWrapper";
+import EmbedVideo from "../../packages/media/EmbedVideo";
+import { Container } from "../../packages/grids/Container";
+import { FAIcon } from "../../packages/icons/FAIcon";
+import HeroQuote from "./HeroQuote";
+import * as styles from "./styles";
+
+/**
+ * Campaign Hero
+ *
+ * @param {object} bkgConfig
+ * Background configuration - could be image or looping video from Vimeo
+ *
+ * @param {object} textConfig
+ * Text configuration - title and intro text for the hero
+ *
+ * @param {string} youtubeId
+ * YouTube ID of the video that will play in a modal
+ *
+ * @param {object} quoteConfig
+ * Quote configuration - fields for the optional quote at the bottom of the hero
+ *
+ * @param {object} bkgImageData
+ * Data from the background image - src and alt text
+ *
+ * @param {object} quoteImageData
+ * Data from the quote image - src and alt text
+ *
+ * @param {string} quoteInternalLinkUrl
+ * The internal link URL for the quotee name if using Matrix asset link
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+
+export default function CampaignHero({
+  bkgConfig,
+  textConfig,
+  youtubeId,
+  quoteConfig,
+  bkgImageData,
+  quoteImageData,
+  quoteInternalLinkUrl,
+}) {
+  // Conditionals for code readability
+  const isBgVideo = bkgConfig?.type === "Video" && bkgConfig?.bkgVideo;
+  const hasQuote = quoteConfig?.include && quoteConfig?.quote;
+
+  /**
+   * The condition to display a pulled left intro.
+   * 1. Quote is not included
+   * 2. Background is an image
+   * 3. No YouTube video to play in a modal
+   */
+  const isIntroPulledLeft =
+    !quoteConfig.include && bkgConfig.type === "Image" && !youtubeId;
```

Comment:
Due to the number of options available, I chatted with Justin that only under these conditions, a pulled left intro is displayed (which is like one of the approved designs on Figma)

Reply:
Thanks for putting the comment in there!